Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md

```diff
@@ -0,0 +1,59 @@
+<h1 align="center" style='color:mediumseagreen;font-weight:bold'>Velaptor Preview Release Notes - v1.0.0-preview.7</h1>
+
+<h2 align="center" style='font-weight:bold'>Quick Reminder</h2>
+
+As with all software, including releases, there is always a chance for issues and bugs.  It is also common to miss changes in the release notes when there are many.  This is even more common in preview releases.
+
+---
+
+<h2 style="font-weight:bold" align="center">New 🎉</h2>
+
+1. Add ability to render rectangles to the scene with the various attributes below  
+   GitHub Issues: [#75](https://github.com/KinsonDigital/Velaptor/issues/75)
+   - Solid/filled rectangles
+   - Empty rectangles
+   - Ability to change the fill color of a filled rectangle
+   - Ability to change the border color of an empty rectangle
+   - Ability to have a horizontal color gradient with a start and end color
+     - This can be applied to filled and empty rectangles
+   - Ability to have a vertical color gradient with a start and end color
+     - This can be applied to filled and empty rectangles
+   - Ability to change the border thickness of an empty rectangle
+   - Ability to set radius of each corner independently
+     - This can be applied to filled and empty rectangles
+2. Loading sounds can not be cached to greatly improve sound loading performance.  
+   GitHub Issues: [#131](https://github.com/KinsonDigital/Velaptor/issues/131)
+3. Loading sounds can now be loaded via a fully qualified file path.  This means you do not have to only load sounds from the content system!!  
+   GitHub Issues: [#128](https://github.com/KinsonDigital/Velaptor/issues/128)
+
+---
+
+<h2 style="font-weight:bold" align="center">Breaking Changes 💣</h2>
+
+1. The class `SystemMonitor` constructor used to require an `IPlatform` type argument but not does not and has no arguments.  
+   GitHub Issues: [#153](https://github.com/KinsonDigital/Velaptor/issues/153)
+
+---
+
+<h2 style="font-weight:bold" align="center">Improvements 🌟</h2>
+
+1. Upgraded all projects in solution to use C# language version **_v10_**  
+   GitHub Issues: [#97](https://github.com/KinsonDigital/Velaptor/issues/97), [#184](https://github.com/KinsonDigital/Velaptor/issues/184)
+2. Upgraded all projects in solution to use **_NET 6.0_**  
```

Comment:
Does the solution need to be named?
Should this be .Net 6.0?

Reply:
It is common to use .NET with all caps.
The solution does not need to be named.  There is only one solution and 90% of the time, projects only have one solution even though you can have more.